lib/resource/Organization.js

```diff
@@ -49,6 +50,32 @@ Organization.prototype.getDefaultGroupStore = function getDefaultGroupStore(/* [
   return this.dataStore.getResource(this.defaultGroupStoreMapping.href, args.options, OrganizationAccountStoreMapping, args.callback);
 };
 
+Organization.prototype.getDefaultAccountStore = function getDefaultAccountStore(/* [options,] callback */) {
+  var args = utils.resolveArgs(arguments, ['options', 'callback'], true);
+  var dataStore = this.dataStore;
+
+  return this.getDefaultAccountStoreMapping({ expand: 'accountStore' }, function (err, accountStoreMapping) {
```

Comment:
`accountStoreMapping` should be `organizationAccountStoreMapping`, as it is that type.